components/Critical/SideProgressBar.tsx

```diff
@@ -1,100 +1,57 @@
 import React from 'react'
-import Box from '@material-ui/core/Box'
-import List from '@material-ui/core/List'
-import ListItem from '@material-ui/core/ListItem'
-import ListItemIcon from '@material-ui/core/ListItemIcon'
-import ListItemText from '@material-ui/core/ListItemText'
-import { withStyles } from '@material-ui/core'
+import {
+  withStyles,
```

Comment:
thank you for cleaning up this janky stuff ;-; !!!
also seems like there might be a few unused imports on this file (and potentially other files haven't checked them yet)